config.tpl.yml

```diff
@@ -13,9 +13,14 @@ use_existing_rg: false
 # If set to true, will disable telemetry for azhop. See https://azure.github.io/az-hop/deploy/telemetry.html.
 #optout_telemetry: true
 
-# Create a log analytics workspace to enable monitoring and alerting
+# A log analytics workspace can be created for monitoring or alerting 
+# Alternatively, you can use an existing workspace. 
+# To use an existing workspace set create to false and specify the resource group the target workspace lives in
 log_analytics:
   create: true
+  # An existing log analytics workspace can be used instead. The name and resource group of the existing workspace will need to be specified.
```

Comment:
Let's use the workspace ID instead of RG and Name as this will allow using WS from another subscription